pyproject.toml

```diff
@@ -38,6 +38,7 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
```

Comment:
you need to add python 3.13 for github actions
I have added it, please rebase

Reply:
Done